app/import.go

```diff
@@ -19,7 +18,7 @@ import (
 
 func stopOnError(err LineImportWorkerError) bool {
 	if err.Error.Id == "api.file.upload_file.large_image.app_error" {
-		mlog.Warn(fmt.Sprintf("Large image import error: %s", err.Error.Error()))
+		mlog.Warm("Large image import error", mlog.Err(err))
```

Comment:
this is failing because the real error is `err.Error`, can you fix that?

Reply:
Also there's a typo. It should be `mlog.Warn`.

Reply:
Also there's a typo. It should be `mlog.Warn`.